src/revlanguage/moves/Move_CharacterHistory.cpp

```diff
@@ -339,6 +582,8 @@ const MemberRules& RevLanguage::Move_CharacterHistory::getParameterRules(void) c
         optionsGraph.push_back( "branch" );
         optionsGraph.push_back( "cladogenetic" );
         optionsGraph.push_back( "cladogenetic2" );
+        optionsGraph.push_back( "tip" );
+        optionsGraph.push_back( "root" );
```

Comment:
GitHub doesn't let me properly highlight lines in this file, but there are a few changes I'd like to see:

- Lines 571--578 were clearly just copied over here by someone reminding themselves of how `OptionRule` works. Let's delete them.
- The `graph` and `proposal` arguments lack argument descriptions: see the empty strings (`""`) on lines 587, 593.
- No need to duplicate the "Inherit weight..." comment, so let's delete lines 595--596.